chef/data_bags/crowbar/template-network.json

```diff
@@ -8,7 +8,9 @@
       "enable_tx_offloading": true,
       "mode": "single",
       "teaming": {
-        "mode": 1
+        "mode": 1,
+        "miimon": 100,
+        "xmit_hash_policy": "layer2"
```

Comment:
I'm 80% sure we want to allow changing these settings on a per-network basis. That's why we have the `team_mode` attribute for networks, that can override the `mode` attribute here.